chapter2/tests/t20/terraform/dev/dev.tfvars

```diff
@@ -0,0 +1,20 @@
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ env=specific configuration variables                                                                             │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+environment = "dev"
+
+
+/* 
+  ┌──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ configuration variables                                                                                    │
+  └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+ */
+
+vpc_tags = {
+  Name = "default"
+}
+
+availability_zones         = ["us-east-2a", "us-east-2b", "us-east-2c"]
```

Comment:
the solution is impossible to maintain with so little vars. Everything is hardcoded - means the maintenance is HELL as you have to modify multiple files and that way you increase the complexity of the modification increasing the chances of the failure === downtime.

Reply:
> the solution is impossible to maintain with so little vars. Everything is hardcoded - means the maintenance is HELL as you have to modify multiple files and that way you increase the complexity of the modification increasing the chances of the failure === downtime.

@dmitry-mightydevops
Done